tap_zendesk/schemas/calls.json

```
@@ -0,0 +1,245 @@
+        "callback": {
+            "type": [
+                "null",
+                "boolean"
+            ]
+        },
+        "callback_source": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "completion_status": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "consultation_time": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "created_at": {
+            "type": [
+                "null",
+                "date"
+            ]
+        },
+        "customer_requested_voicemail": {
+            "type": [
+                "null",
+                "boolean"
+            ]
+        },
+        "default_group": {
+            "type": [
+                "null",
+                "boolean"
+            ]
+        },
+        "direction": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "duration": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "exceeded_queue_time": {
+            "type": [
+                "null",
+                "boolean"
+            ]
+        },
+        "hold_time": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "id": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "ivr_action": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "ivr_destination_group_name": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "ivr_hops": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "ivr_routed_to": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        "ivr_time_spent": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "minutes_billed": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "not_recording_time": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
+        "outside_business_hours": {
+            "type": [
+                "null",
+                "boolean"
+            ]
+        },
+        "overflowed": {
+            "type": [
+                "null",
+                "boolean"
+            ]
+        },
+        "overflowed_to": {
+            "type": [
+                "null",
+                "boolean"
+            ]
+        },
+        "phone_number": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
```

```suggestion
        },
        "phone_number_id": {
            "type": [
                "null",
                "integer"
            ]
        },
```

Waiting on upstream: https://github.com/facetoe/zenpy/pull/454#issuecomment-710077200